app/javascript/packages/analytics/index.spec.ts

```diff
@@ -26,44 +40,71 @@ describe('trackEvent', () => {
     });
 
     context('no payload', () => {
-      it('fetches and resolves to undefined', async () => {
-        const result = await trackEvent('name');
+      it('calls sendBeacon and resolves to undefined', async () => {
+        const result = trackEvent('name');
 
         expect(result).to.be.undefined();
-        expect(global.fetch).to.have.been.calledWith(
-          endpoint,
-          sandbox.match({
-            body: '{"event":"name"}',
-            headers: { 'Content-Type': 'application/json' },
-            method: 'POST',
-          }),
-        );
+
+        expect(global.navigator.sendBeacon).to.have.been.calledOnce();
+
+        const [actualEndpoint, data] = (global.navigator.sendBeacon as SinonStub).firstCall.args;
+        expect(actualEndpoint).to.eql(endpoint);
+        expect(data).to.have.property('type').eql('application/json');
+
+        expect(await blobTextContents(data)).to.eql('{"event":"name"}');
+      });
+
+      it('does not call fetch', () => {
```

Comment:
I think if the implementation doesn't involve `fetch` anymore, we could remove any references to fetch in this test file.

I suppose an argument could be made that the function might _imply_ that it calls `fetch` and we'd want some assurance that it doesn't. Personally, I still don't think it's necessary.